Packs/CommonScripts/Scripts/script-DeleteContext.yml

```diff
@@ -54,6 +48,39 @@ script: >
               keysToKeepObj[keysToKeep[i]] = value;
           }
       }
+      return keysToKeepObj;
+  }
+
+  function createContext(keysToKeepObj) {
+      context = {};
+      for (var key in keysToKeepObj) {
+          newKey = key.replace(/.\[\d+\]/g, '');
+
+          while(newKey != newKey.replace(/.\[\d+\]/g, '')){
+              newKey = newKey.replace(/.\[\d+\]/g, '')
+          }
+
+          if (context[newKey]) {
```

Comment:
what if the value was false? or 0?
I think is better to check for `undefined`

Reply:
changed to "in" :
![image](https://user-images.githubusercontent.com/50294648/107129025-272f6780-68cb-11eb-82b8-498283c27ab2.png)